developer/src/kmc-package/src/kmp-compiler.ts

```diff
@@ -220,6 +220,8 @@ export default class KmpCompiler {
    * @param kmpJsonData - The kmp.json Object
    */
   public buildKmpFile(kpsFilename: string, kmpJsonData: KmpJsonFile): Promise<any> {
+    const zip = JSZip();
```

Comment:
Good catch! That would have been _nasty_ to send out! Shame we had to do this fix in the merge PR but all good.

Reply:
Yes, let's just say it felt less and less like a merge PR as time went on… in any event, that's why keeping in sync is important.